launchable/commands/record/case_event.py

```diff
@@ -76,10 +78,50 @@ def path_canonicalizer(test_path: TestPath) -> TestPath:
 
             return test_path
 
+        def stdout(case: TestCase) -> str:
+            """
+            case for:
+                <testcase>
+                <system-out>...</system-out>
+                </testcase>
+            """
+            if case.system_out is not None:
+                return case.system_out
+
+            return ""
+
+        def stderr(case: TestCase) -> str:
+            """
+            case for:
+                <testcase>
+                <system-err>...</system-err>
+                </testcase>
+            """
+            if case.system_err is not None:
+                return case.system_err
+
+            """
+            case for:
+                <testcase>
+                <failure message="...>...</failure>
+                </testcase>
+            """
+            stderr = ""
+            for result in case.result:
+                if any(isinstance(result, r) for r in POSSIBLE_RESULTS):
+                    if result.message and result.text:
+                        stderr = stderr + result.message + "\n" + result.text
+                    elif result.message:
+                        stderr = stderr + result.message
+                    elif result.text:
+                        stderr = stderr + result.text
```

Comment:
[question]
Is it correct that this is appending result.message and result.text if they exist?
Let's say you have 3 results, and they all have result.message, like "aa", "bb" and "cc" respectively.
Does the result be like this below?
1st stderr result: aa
2nd stderr result: aabb
3rd stderr result: aabbcc
Or, do you always have line breaks after message that you don't have to take care of separating messages?

Reply:
> Or, do you always have line breaks after message that you don't have to take care of separating messages?

Yes, I checked some test runner reports, then reports are usually put "\n" at last. 
So, I didn't care it for now. If we faced some problems, I'll fix this then.

Reply:
If that is the case, why do you need a line break in `result.message + "\n" + result.text` ?

Reply:
Sorry, I misunderstood that you mention only result.text. 

Test runner which I checked usually produces stderr data like below
```
<testcase>
  <failure message="FAIL_RESULT"> STACK TRACE ... </failure>
</testcase>
```

We can get message data (="FAILE_RESULT") use by result.message method. Also, we can get failure's text(=STACK TRACE...) use by result.text.

Usually, message data doesn't include "\n", so I join message and text with "\n".

After writing the comment, I realized I should change to `stderr = stderr + result.message + "\n"`.
So, I'll change only it.